src/test/scala/tech/cryptonomic/conseil/tezos/TezosPlatformDiscoveryOperationsTest.scala

```diff
@@ -187,6 +187,9 @@ class TezosPlatformDiscoveryOperationsTest
         sut.getTableAttributes("operations").futureValue shouldBe
           Some(
             List(
+              Attribute("branch", "Branch", DataType.String, None, KeyType.NonKey, "operations"),
```

Comment:
Possibly another PR

What do you think about using an order-independent verification method like doing
```scala
sut.getTableAttributes(...).futureValue.value should contain theSameElementsAs List(...)
```
, so that random shuffling of the attributes output based on slick's definition will not break the test anymore?